firewall/firewall_configuration_provider.go

```diff
@@ -56,7 +56,7 @@ func (firewall *Firewall) update(configuration []byte) error {
 			}
 
 			firewallCheckers = append(firewallCheckers, whitelistHandler)
```

Comment:
why not add handler here? instead of second loop after this

Reply:
Thanks. Agree